File: DevSlopes_Tutorials_Apps/iOS - Main Courses/ObjC for Swift Devs/Section 1/7 - GettersSetters/GettersSetters/ViewController.m
//
//  ViewController.m
//  GettersSetters
//
//  Created by Jakub Majewski on 16.08.2017.
//  Copyright © 2017 Jakub Majewski. All rights reserved.
//

#import "ViewController.h"
#import "Vehicle.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    Vehicle *car = [[Vehicle alloc]init];
    car.odometer = -100;
    
    NSLog(@"Odomer: %lu", car.odometer);
    
    car.model =@"BMW 328i";
    
    NSLog(@"I drive a: %@", car.model);
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: DevSlopes_Tutorials_Apps/iOS - Main Courses/iOS11 From Beginner to Paid Professiona /Section 2 - Learn to Code With Swift 4 /ColorMagic/ColorMagic/ViewController.swift
//
//  ViewController.swift
//  ColorMagic
//
//  Created by Jakub Majewski on 24.09.2017.
//  Copyright © 2017 Jakub Majewski. All rights reserved.
//

import UIKit

class ViewController: UIViewController, ColorTransferDelegate {

    @IBOutlet weak var pickedColor: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    func userDidChoose(color: UIColor, withName colorName: String) {
        self.view.backgroundColor = color
        self.pickedColor.text = colorName
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        if segue.identifier == "presentColorPickerVC" {
            guard let colorPickerVC = segue.destination as? PickColorViewController else { return }
            colorPickerVC.delegate = self
        }
    }

    
}

